autofill/autofill-impl/src/main/res/values/donottranslate.xml

```diff
@@ -15,5 +15,10 @@
   -->
 
 <resources>
-
+    <!-- Sync settings screen -->
+    <plurals name="syncCredentialInvalidItemsWarning" instruction="%1$ represents a site name or url, %2$d is the number of other invalid sites">
+        <item quantity="one">Your password for %1$s can’t sync because one of its fields exceeds the character limit.\n\n</item>
+        <item quantity="other">Your passwords for %1$s and %2$d other sites can’t sync because some of their fields exceed the character limit.\n\n</item>
```

Comment:
nit: it's a little weird to say

- "Your passwords for ABC and 1 other sites" (plural) but super minor so just FYI

Reply:
yeah I can understand what you mean. Gonna skip this one for now and revisit if there's internal/external feedback.